src/typhoon_automator/simulation.py

```diff
@@ -25,10 +31,17 @@ def __init__(
         
         self._schedule = EventSchedule()
 
+        self.stop_signal = False
+        self.start_time: datetime = None
+        self.stop_time: datetime = None
+
```

Comment:
I missed this in the last repo, so now's our chance to fix it:  Python doesn't have public/private class variables, so variables meant to be private are prefixed with an underscore (and public without an underscore).  `stop_signal`, `start_time`, and `stop_time` are all meant to be private and should have the underscore prefix.  If you can make this change (e.g. `self._stop_signal` and likewise), that'd be great!